walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/TimeRange.java

```diff
@@ -46,35 +46,70 @@ public int compare(TimeRange a, TimeRange b) {
     }
   };
 
-  private final int start;
-  private final int duration;
+  private int start;
+  private int duration;
+  private boolean isSchedulable;
+  private boolean isValidRange;
 
   private TimeRange(int start, int duration) {
     this.start = start;
     this.duration = duration;
+    this.isSchedulable = true;
+    this.isValidRange = true;
+  }
+
+  public TimeRange() {
+      this.start = 0;
+      this.duration = 0;
+      this.isSchedulable = false;
+      this.isValidRange = false;
   }
 
   /**
    * Returns the start of the range in minutes.
    */
-  public int start() {
+  public int getStart() {
     return start;
   }
 
+  public void setStart(int start) {
+    this.start = start;
+  }
+
   /**
    * Returns the number of minutes between the start and end.
    */
-  public int duration() {
+  public int getDuration() {
     return duration;
   }
 
+  public void setDuration(int duration) {
+    this.duration = duration;
+  }
+
   /**
    * Returns the end of the range. This ending value is the closing exclusive bound.
    */
   public int end() {
     return start + duration;
   }
 
+  public boolean isSchedulable() {
```

Comment:
same. though it's probably OK for the accessors to be foo() instead of getFoo(). Up to you, but the important part is to be consistent.